mkdocs.yml

```diff
@@ -53,6 +53,7 @@ extra:
 nav:
   - Introduction: index.md
   - Getting Started: getting-started.md
+  - Letting Go: letting-go.md
```

Comment:
"Letting Go" will show up in menu? I don't think that's a good title. "Migration to Platform" or like the title of this page "Converting from Data Hub to IML"